### DISCUSSION TITLE
Add commit history for files

### DISCUSSION THREAD

**Discussion by sethfri** [community/community#20793] [Mobile]
Currently, when viewing files in a repo in the iOS app, there is no way to view the commit history or the blame for that file. This makes it difficult to try to figure out when a file was last changed and what changes have been made to that file recently.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.72.0 (72265070)
Device: iPhone 13 Pro (iOS 15.5)
</details>

**Comment by iskae** [comment]
Hi @sethfri 👋 

Thanks a bunch for reaching out and for your feedback ❤️

It looks like we already have an internal tracking issue for your request, therefore I will be upvoting and adding your feedback in that issue too 👍 Please stay tuned for our future releases 🙇 

Cheers!
